app/code/Magento/Bundle/view/base/web/js/price-bundle.js

```diff
@@ -375,6 +375,17 @@ define([
         var tiers = optionConfig.tierPrice,
             magicKey = _.keys(oneItemPrice)[0],
             lowest = false;
+        
+        //tiers is undefined when options has only one option 
+		if (undefined == tiers) {
```

Comment:
Hi @adarshkhatri. Thanks for collaboration. 
According to [Magento Technical Guidelines](https://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html) we should use `===` instead of `==`. 

In this case, you can make only a simple check `if (tiers)` or if you need to compare value directly to undefined you can use underscore.js method _.isUndefined

Reply:
oh! thanks, will update accordingly.